modules/apps/object/object-test-util/src/main/java/com/liferay/object/test/util/ObjectRelationshipTestUtil.java

```diff
@@ -37,9 +37,21 @@ public static ObjectRelationship addObjectRelationship(
 			ObjectDefinition objectDefinition2, long userId)
 		throws PortalException {
 
+		return addObjectRelationship(
+			objectRelationshipLocalService, objectDefinition1,
+			objectDefinition2, userId, 0);
+	}
+
+	public static ObjectRelationship addObjectRelationship(
+			ObjectRelationshipLocalService objectRelationshipLocalService,
+			ObjectDefinition objectDefinition1,
+			ObjectDefinition objectDefinition2, long userId,
+			long parameterObjectFieldId)
+		throws PortalException {
```

Comment:
I think we need to sort `long userId` with `long parameterObjectFieldId`

Reply:
I'm following the existing pattern in this class